### DISCUSSION TITLE
Cannot unlock IOS device Original error: Unexpected data: {"Error":"PasswordProtected","Request":"StartService"}

### DISCUSSION THREAD

**Discussion by wrathmelior** [appium/appium#16579] [Q&A]
I am getting error Original error: Unexpected data: {"Error":"PasswordProtected","Request":"StartService"} after adding a pin (required by google for use with google SSO) to my (real) IOS device.

I am using the following desired capabilities but appium seems to be ignoring them. 
  "appium:unlockType": "pin",
  "appium:unlockKey": "111111"

**Comment by KazuCocoa** [comment]
It seems like it is the same as:

> iPhone/iPad real devices https://github.com/appium/appium/issues/15898#issuecomment-927340411 when they start a XCTest session since iOS/iPadOS 15. Disabling passcode/touch id in the device preference allows to workaround the behaviour above.

https://github.com/appium/appium-xcuitest-driver#known-problems

So, Appium/WDA needs a XCTest session. the session requires to unlock the device in the OS layer before starting.
